packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/index.js

```diff
@@ -43,6 +63,56 @@ export default function SidebarNavigationScreenNavigationMenu() {
 
 	const menuTitle = navigationMenu?.title?.rendered || navigationMenu?.slug;
 
+	const handleSave = async () => {
+		saveEditedEntityRecord( 'postType', postType, postId );
+		setOpen( false );
+		createSuccessNotice( 'Renamed Navigation menu', {
+			type: 'snackbar',
+		} );
+	};
+	const handleChange = ( title ) =>
+		editEntityRecord( 'postType', postType, postId, { title } );
+	const handleDelete = () => {
+		deleteEntityRecord( 'postType', postType, postId, { force: true } );
+		createSuccessNotice( 'Deleted Navigation menu', {
+			type: 'snackbar',
+		} );
+		goTo( '/navigation' );
+	};
+	const handleDuplicate = async () => {
+		const savedRecord = await saveEntityRecord( 'postType', postType, {
+			title: menuTitle + __( ' (Copy)' ),
+			content: navigationMenu?.content?.raw,
+			status: 'publish',
+		} );
```

Comment:
```suggestion
		const savedRecord = await saveEntityRecord( 'postType', postType, {
			title: menuTitle + __( ' (Copy)' ),
			content: navigationMenu?.content?.raw,
			status: 'publish',
		} ).catch()
```

We should add error handling as a catch here. The user needs to know if the action failed.

Reply:
✅